src/App.js

```diff
@@ -186,7 +200,9 @@ function App() {
                 ) : null}
                 <GuestNavItems>
                   <NavItem to="/support-the-project/">☆ תמיכה בפרוייקט</NavItem>
-                  <NavItem to="/add-protest/">+ הוספת הפגנה</NavItem>
+                  <NavItem to="/" onClick={() => (window.location = getAddProtestButtonLink(state.user))}>
```

Comment:
Using `window.location` is making the browser change a page.  
Use  the `<Link>` component to keep the visitor in the app using react router.